src/utils/url.test.ts

```diff
@@ -7,14 +7,14 @@ const skylink = "XABvi7JtJbQSMAcDwnUnmp2FKDPjg8_tTTFP4BwMSxVdEg";
 const skylinkBase32 = "bg06v2tidkir84hg0s1s4t97jaeoaa1jse1svrad657u070c9calq4g";
 
 describe("addUrlQuery", () => {
-  const parts: Array<[string, Record<string, unknown>, string]> = [
+  const parts: Array<[string, { [key: string]: string | undefined }, string]> = [
```

Comment:
```suggestion
  const parts: Array<[string, { [key: string]: string | number | boolean }, string]> = [
```

we should keep supporting all basic types - namely numbers and booleans